.github/workflows/windows.yml

```diff
@@ -0,0 +1,76 @@
+on:
+  push:
+    branches:
+      - clean-meson-windows
+  pull_request:
+    branches:
+      - clean-meson-windows
+
+jobs:
+  test:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Choco python
+        uses: crazy-max/ghaction-chocolatey@v1
+        with:
+          args: install python --version=3.9.9 --no-progress
+      - name: set-python-path
+        run: |
+          # Install path, culled from output of Choco install above.
+          $py_dir = 'C:\hostedtoolcache\windows\Python\3.9.9\x64'
+          echo "$py_dir;$py_dir\Scripts;" >> $env:GITHUB_PATH
+      - name: show-python-version
+        run: |
+          python --version
+      - name: install-rtools
+        run: |
+          choco install rtools --no-progress
+          echo "c:\rtools40\ucrt64\bin;" >> $env:GITHUB_PATH
+      - name: show-gfortran
+        run: |
+          gcc --version
+          gfortran --version
+      - name: pip-packages
+        run: |
+          pip install numpy cython pybind11 pythran meson ninja
+      - name: openblas-libs
+        run: |
+          choco install unzip
+          pip install gdown
+          gdown --id 1G3jwxMY4ZVRqvz6owl0kmw_Innmr4ZnV
```

Comment:
This could use a comment about this download containing only openblas and how it was built.

Reply:
Yes - I'll do one - thanks.  I'll also suggest this build procedure over at openblas-libs.